ChangeLog.md

```diff
@@ -1,4 +1,108 @@
-﻿## 2017.08.07 - Version 4.3.0
+﻿## 2017.08.10 - Version 4.3.1
```

Comment:
@markcowl looks like the `## 2017.08.07 - Version 4.3.0` header will be removed and replaced with the new one for 4.3.1. I wasn't sure if this was intended, or if it was an accident.

This should get figured out when removing the duplicate change log entries in this file.